packaging/docker/Dockerfile

```diff
@@ -32,7 +32,9 @@ ARG FDB_VERSION
 ARG FDB_WEBSITE=https://www.foundationdb.org
 
 WORKDIR /var/fdb/tmp
-RUN wget $FDB_WEBSITE/downloads/$FDB_VERSION/linux/fdb_$FDB_VERSION.tar.gz && \
+ADD website /mnt/website
```

Comment:
What is this line adding?

Reply:
This line is only needed when using the file:// scheme rather than http(s)://. It would allow the user to position the required files within the build docker.

Reply:
So this is pulling in a "website" folder from the build environment? What does it do if that folder is not present?